pkger/models.go

```diff
@@ -317,7 +317,7 @@ func (d DiffLabel) IsNew() bool {
 }
 
 func (d DiffLabel) hasConflict() bool {
-	return d.IsNew() || d.Old != nil && *d.Old != d.New
+	return !d.IsNew() && d.Old != nil && !reflect.DeepEqual(*d.Old, d.New)
```

Comment:
reflect.DeepEqual is a little expensive. Since their type is same, just make a simple func to compare both Color and desc.

Reply:
funny enough, i was just changing this :-), good catch 👌

Reply:
since the types are struct values, they can be compared directly and go will resolve the equality 👍